products/pluto/metadata.yml

```diff
@@ -1,675 +1,1705 @@
+      - [07, Transportation & Utility]
+      - [08, Public Facilities & Institutions]
+      - [09, Open Space & Outdoor Recreation]
+      - [10, Parking Facilities]
+      - [11, Vacant Land]
+
+  - name: easements
+    display_name: easements
+    description: |
+      The number of unique easements on the tax lot.
+
+      PTS contains a record for each easement. NUMBER OF EASEMENTS is calculated by counting the number of unique PTS easement records for the tax lot.
+
+      If the number of easements is zero, the tax lot has no easements.
+    data_source: Department of Finance – Property Tax System (PTS)
+    data_type: text
+    readme_data_type: Numeric - 2 digits (99)
+    example: "0"
+
+  - name: ownertype
+    display_name: ownertype
+    description: |
+      A code indicating type of ownership for the tax lot.
+
+      Only one data source is used per tax lot.
+
+      The COLP file, which contains more accurate and specific type of city ownership data than PTS, is used when data is available for that lot. Codes C, M, O, P are derived from COLP.
+
+      If the tax lot is not in COLP, PTS is checked to see if the lot’s EXEMPT TOTAL VALUE equals its ASSESSED TOTAL VALUE. If the two values are the same, the lot is given a code of X. Otherwise the tax lot is not given any TYPE OF OWNERSHIP CODE.
+
+      OWNER NAME should be referenced to verify type of ownership, particularly when it’s important to distinguish between state, federal, and public authority ownership.
+    data_source: |
+      Department of City Planning - City Owned and Leased Properties (COLP)
+      Department of Finance - Property Tax System (PTS)
+    data_type: text
+    readme_data_type: Alphanumeric - 1 character
+    example: X
+    values:
+      - [C, "City ownership"]
+      - [X, "Fully tax-exempt property that may be owned by the city, state, or federal government; a public authority; or a private institution"]
+      - [O, "Other – owned by either a public authority or the state or federal government"]
+      - [P, "Private ownership"]
+      - [M, "Mixed city & private ownership"]
+#      - [null, Unknown (usually private ownership)] # TODO: implement nulls for values. I didn't realize this use-case
+
+  - name: ownername
+    display_name: ownername
+    description: |
+      The name of the owner of the tax lot.
+
+      For publicly owned tax lots, owner names have been normalized. For example, “NYC PARKS”, “PARKS DEPARTMENT”, and “PARKS AND RECREATION (GENERAL)” have been changed to “NYC DEPARTMENT OF PARKS AND RECREATION”.
+
+      If OWNER NAME is normalized, DCPEdited is set to “1”. (see CHANGED BY DCP).
+    data_source: |
+      Department of Finance - Property Tax System (PTS)
+      Department of City Planning – [PLUTO_input_research.csv](https://github.com/NYCPlanning/db-pluto/blob/dev/pluto_build/data/pluto_input_research.csv), field ownername
+    # TODO: should we have elements like links as markdown, or html?
```

Comment:
Probably depends if we can transfer the markdown format to the generated pluto metadata? i.e. link name + link

Reply:
markdown could be nice, but html might be more compatible for pushing straight to destinations and for generating docs

Reply:
yeah, very much an open question. Happy to hear thoughts 🙂